scripts/mktest.py

```diff
@@ -271,6 +272,9 @@ def naturalearth_test(args):
     ne_test_parser.add_argument(
         '--ne-id', type=int, default=1,
         help='Natural Earth ID to use.')
+    ne_test_parser.add_argument(
+        '--layer-name', default='pois',
```

Comment:
Nit: Not many NE features ending up in `pois` layer, maybe `places` is better?

Reply:
Good point. Fixed in f585eff.